@uportal/esco-content-menu/src/components/ActionFavorites.vue

```diff
@@ -66,7 +67,7 @@ export default {
     favoriteMessage() {
       return this.translate(
         this.isFavorite ? 'message.favorites.remove' : 'message.favorites.add'
-      );
+      ).replace('{}', this.portletDesc.title);
```

Comment:
> I don't understand the goal of this change ???

The goal of this change is to provide a message to the title on lines 7 and 10 as to what clicking the star will do. 

> You are setting i18n message to empty ?

It may be empty since the French language strings do not exist at the moment

Reply:
Sorry I didn't understood, but finally I realize that's ok with the new messages !